### ISSUE TITLE
Parsing of Dates

### ISSUE THREAD

**Issue by ghacupha** [ozlerhakan/poiji#9]
Could we also add support for locales? I realized with the dates, it seems to work when I use the US format and it is precisely like this : "MM/dd/yy"
If I specify "MM/dd/yyyy" it will not work, if I use the UK format it will not work

**Comment by ozlerhakan** [comment]
Hi @ghacupha ,

Have you looked at the test?

https://github.com/ozlerhakan/poiji/blob/fbeb77d29cfa02511c48b74f54f937157654679a/src/test/java/com/poiji/deserialize/DerializersExtendedTest.java#L53

Could you explain your case?

**Comment by ghacupha** [comment]
Am sorry I have not been able to submit the tests. Been swamped. Let me do that over this weekend, or in the course of next week

**Comment by ozlerhakan** [comment]
no problem, take your time @ghacupha

**Comment by ghacupha** [comment]
I looked at the test and added my own using a test using the simpleDateFormatter to check if the the dates were the same. Coming soon to a pull request near you...
I was wrong. The dates were reading just fine, only that I had mismatched the pattern at first, and assumed that it was a locale specification problem, but java.util.Date does not have to have locale. So I think the date thing is working just fine
